src/components/pages/variable.tsx

```diff
@@ -1,10 +1,46 @@
-import '../../styles/variable.scss';
+import robot1 from '../../assets/robots/pg2/robot1pg2.svg';
+import robot2 from '../../assets/robots/pg2/robot2pg2.svg';
+import robot3 from '../../assets/robots/pg2/robot3pg2.svg';
+import robot4 from '../../assets/robots/pg2/robot4pg2.svg';
+import {lineOfCode, CodeFormat}  from  '../shared/codeFormat';
+import {Color} from '../shared/constants';
+
 function Variable(): JSX.Element {
+
+  const code: lineOfCode[] = [
+    [
+      { ['num_of_antennas']: Color.White },
+      { [' = ']: Color.Orange },
+      { ['0']: Color.Blue },
+    ],
+    [
+      { ['has_mouth']: Color.White },
+      { [' = ']: Color.Orange },
+      { ['False']: Color.Blue }    ],
+    [
+      { ['color']: Color.White },
+      { [' = ']: Color.Orange },
+      { ['"blue"']: Color.White }    ],
```

Comment:
Just noticed that string values are in white, would blue make more sense since number and bool values are blue and variable names are white, wondering what your thoughts are.

Reply:
I'm going off the protocol on [https://www.onlinegdb.com/](https://www.onlinegdb.com/). I'm assuming it's because strings are not primitives while number/bool values are. But if you think it might confuse the kids, I can change it your call.